src/app.css

```diff
@@ -0,0 +1,22 @@
+@import url("styles/colors.css");
```

Comment:
please be consistent with your whitespace. I'll assign myself a task for linting CSS